cmake/FreecivInstall.cmake

```diff
@@ -127,25 +127,23 @@ elseif(WIN32)
   # The Visual Studio generator places all files and associated DLL libraries
   #  into a build directory. So we just grab those for install.
   install(
-    DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/
+    DIRECTORY ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/debug/bin/
```

Comment:
Does this still work if you ask for a release build instead of a debug build?

Reply:
Yes it does.

Reply:
Strange, I wouldn't expect a `release` build to put files in a `debug` folder (did you start the release build in a fresh build folder?).

Reply:
LoL. Its Visual Studio + vcpkg + Windows craziness